parent/pom.xml

```
@@ -157,10 +157,13 @@
     <jetty-http.version>9.4.11.v20180605</jetty-http.version>
     <jetty-server.version>9.4.11.v20180605</jetty-server.version>
     <junit.version>4.13-beta-3</junit.version>
+    <junit-jupiter.version>5.4.2</junit-jupiter.version>
```

@srnagar observed that the latest version is 5.5.2. Here's a suggested fix for this:
```suggestion
    <junit-jupiter.version>5.5.2</junit-jupiter.version>
```